content/millennium/overview.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Overview
+layout: api
+---
+
+# Overview
+
+* TOC
+{:toc}
+
+## Supported Versions
+
+Cerner’s implementation currently supports both the R4 First Normative Content (4.0.1) version and DSTU2 Final (1.0.2) version of the HL7 FHIR® standard. Cerner’s implementation of the R4 version is ongoing and new resources and actions will be added over time.
+
+## Determining Which Version You Should Use
+
+Determining which version you should use will depend upon your use case and, in general, if the R4 implementation meets all of your requirements, you should target your development against the R4 version. To determine if the R4 version meets your requirements, you need to consider supported resources and actions, patient access, and technical requirements. In general, there are more capabilities such as create and modify in R4 that will not be patched back to DSTU2.
+
+Cerner’s R4 implementation currently does not support patient access, nor is it certified for HIT 2015 Edition Cures Act (formerly Meaningful Use 3). Cerner will be adding support for patient access and enhancing R4 APIs over the coming months in time for clients to certify. Cerner’s R4 implementation will achieve a like-for-like implementation of all resources and actions currently supported by DSTU2. Existing DSTU2 apps will eventually need to be migrated to our R4 implementation. For this reason, we highly encourage development against R4 whenever possible.
+
+## Supported Resources Between Versions
+
+To determine what resources and actions are supported by our DSTU2 and R4 implementations you can browse our documentation for both DSTU2 and R4 and compare between each version. You can also compare the Conformance (DSTU2) and CapabilityStatement (R4) for the list of supported resources and actions.
```

Comment:
another small nit: Could you add new lines roughly after 120 characters for each paragraph?  It'll make it look cleaner and easier to read when opening it in editors

Reply:
even no line breaks is fine in my opinion

Reply:
Since there are other files with line breaks and they were already committed, I left them in.